lib/id3c/cli/command/etl/manifest.py

```diff
@@ -95,55 +95,71 @@ def etl_manifest(*, db: DatabaseSession):
         with db.savepoint(f"manifest record {manifest_record.id}"):
             LOG.info(f"Processing record {manifest_record.id}")
 
-            # Convert sample barcode to full identifier, ensuring it's
-            # known and from the correct identifier set.
-            sample_barcode = manifest_record.document.pop("sample")
-            sample_identifier = find_identifier(db, sample_barcode)
+            # When updating an existing row, update the identifiers
+            # only if the record has both the 'sample' and
+            # 'collection' keys.
+            should_update_identifiers = "sample" in manifest_record.document \
+                and "collection" in manifest_record.document
 
-            if not sample_identifier:
-                LOG.warning(f"Skipping sample with unknown sample barcode «{sample_barcode}»")
-                mark_skipped(db, manifest_record.id)
-                continue
+             # Sample collection date
+            collected_date = manifest_record.document.pop("date", None)
```

Comment:
I think we still want to keep `date` in the `sample.details` so that we can easily compare the `collected` column and the aliquot manifest date ingested into `details.date`. There are still existing differences that we need to investigate (see [Trello card](https://trello.com/c/XFOlT3JC/738-investigate-the-discordance-between-samplecollected-and-details%E2%86%92date))

Reply:
I agree with Jover here.

Reply:
Sounds good. This will duplicate that data, but it's handy for backwards compatibility.
Fixed.